scss/components/button.scss

```diff
@@ -607,4 +608,4 @@ a.#{$block} {
       }
     }
 }
-/* stylelint-enable */
\ No newline at end of file
+/* stylelint-enable */
```

Comment:
Changes in this file should be reverted.

Reply:
The changes were reverted. I do not see them anymore in the files changed.